doc: Update docs with nRF9151 where the support is added (#14127)
nrfconnect/sdk-nrf: samples/net/http_server/README.rst

If you have issues with connectivity on nRF91 Series devices, see the `Cellular Monitor`_ documentation to learn how to capture modem traces to debug network traffic in Wireshark.

> Modem traces can be enabled by providing a snippet with the west build command using the ``-S`` argument:
> 
> The following is an example for the nRF9161 DK:

Reviewer: ```suggestion
Modem traces can be enabled by providing a snippet with the west build command using the ``-S`` argument as shown in the following example for nRF9161 DK:

```
